index.js

```diff
@@ -581,15 +581,17 @@ var torrentStream = function(link, opts, cb) {
 		if (engine.bitfield) wire.bitfield(engine.bitfield);
 	});
 
-	swarm.pause();
+	var pauseSwarm = true;
 
 	if (link.files && engine.metadata) {
 		swarm.resume();
+		pauseSwarm = false;
 		ontorrent(link);
 	} else {
 		fs.readFile(torrentPath, function(_, buf) {
 			if (destroyed) return;
 			swarm.resume();
+			pauseSwarm = false;
```

Comment:
But again, I think this line: ``swarm.pause()`` in ``torrent-stream`` doesn't do anything, because in both ``if`` and ``else`` that come right after it, ``swarm.resume()`` is called.

I tested ``torrent-stream`` with that line removed and everything works as it should, it looks like it is there as a fallback if ``swarm.resume()`` isn't triggered, but I cannot see how it could ever not be triggered.